camdecmps/procedures/copy_monitor_plan_from_workspace_to_global.sql

```diff
@@ -32,11 +32,25 @@ BEGIN
 		AND stack_pipe_id IS NOT NULL
 	) INTO stackPipeIds;
 
+    WITH mp_timeframe as (
+        SELECT
+            brp.end_date AS begin_date,
+            erp.begin_date AS end_date
+        FROM camdecmpswks.monitor_plan mp
+        JOIN camdecmpsmd.reporting_period brp ON mp.begin_rpt_period_id = brp.rpt_period_id
+        JOIN camdecmpsmd.reporting_period erp ON mp.end_rpt_period_id = erp.rpt_period_id
```

Comment:
This should be a left join unless the MP are never active (End Report Period Id is not null).

Reply:
Good catch! I just pushed that change.